packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -127,6 +128,10 @@ export class DataStores implements IDisposable {
             eventName: "ContainerLoadStats",
```

Comment:
Same suggestion as before - can we leverage ...this.metadata? Is it too much data? Or not right types?

Reply:
that wouldn't be the right type. for logging, it will have to be the individual properties